resources/docker-compose/apim/conf/deployment.toml

```diff
@@ -56,7 +56,7 @@ provider = "wso2"
 display_in_api_console = true
 description = "This is a hybrid gateway that handles both production and sandbox token traffic."
 show_as_token_endpoint_url = true
-service_url = "https://localhost:${mgt.transport.https.port}/services/"
+service_url = "https://apim:${mgt.transport.https.port}/services/"
```

Comment:
I think this is not used anymore in APIM side. Was there a reason for changing this?

Reply:
I see.. There was no specific reason. I changed it to be similar to configs in the tests we used. In the tests, it kept failing when this was localhost.